src/components/CompareResults/SubtestsResults/SubtestsResultsTable.tsx

```diff
@@ -222,8 +221,6 @@ function SubtestsResultsTable({
           rowGridTemplateColumns={rowGridTemplateColumns}
         />
       ))}
-
-      {processedResults.length == 0 && <NoResultsFound />}
```

Comment:
I think it would be good to keep that here, for when the user search or filter filters everything out.